packages/driver/src/multi-domain/unsupported_apis.ts

```diff
@@ -0,0 +1,18 @@
+import $errUtils from '../cypress/error_utils'
+
+export const handleUnsupportedAPIs = (Cypress: Cypress.Cypress) => {
+  // outlaw the use of `route` and `server` within the multi-domain context and Cypress.Server.* configurations
+  // @ts-ignore
+  cy.route = () => $errUtils.throwErrByPath('switchToDomain.route.unsupported')
+  // @ts-ignore
+  cy.server = () => $errUtils.throwErrByPath('switchToDomain.server.unsupported')
```

Comment:
We should pass in `cy` here instead of using the global.

Reply:
whoops 😅 . I think we need both for the big namespaced APIs as well?

Reply:
I passed in `cy` as an additional arg in a329ab1524

Reply:
Both as in `cy` and `Cypress`? Yeah, we should still be passing in `Cypress` as well.